docs/io/unified.rst

```diff
@@ -272,10 +272,16 @@ file that has multiple datasets, use *both* the ``overwrite=True`` and
 
 If the metadata of the table cannot be written directly to the HDF5 file 
 (e.g. dictionaries), or if you want to preserve the units and description
-of tables and columns, use using ``serialize_meta=True``::
+of tables and columns, use ``serialize_meta=True``::
 
     >>> t.write('observations.hdf5', path='updated_data', serialize_meta=True)
- 
+
+The way serialized meta are saved in the HDF5 dataset have changed in Astropy 2.0.
```

Comment:
Say `was changed in Astropy 3.0`.  (And note this PR won't get in to 2.0 unfortunately, a bit too late).